apertium-es-pt.es-pt.crx

```diff
@@ -1,24 +1,35 @@
 <?xml version="1.0"?>
 <capitalization>
   <rules>
-    <rule weight="0.1">
-      <match select="aa"/>
-    </rule>
-    <rule weight="1.0">
+    <rule weight="3.0">
       <or>
-        <match lemma="iPhone" select="dix"/>
-        <match lemma="iPod" select="dix"/>
+        <begin/>
+        <match tags="sent"/>
       </or>
+      <match select="Aa"/>
     </rule>
-    <rule weight="0.9">
+    <rule weight="2.0">
       <match tags="np.*" select="Aa"/>
     </rule>
     <rule weight="1.0">
       <or>
-        <begin/>
-        <match tags="sent"/>
+        <match lemma="natal" select="Aa"/>
+        <match lemma="páscoa" select="Aa"/>
+        <match lemma="carnaval" select="Aa"/>
```

Comment:
could you explain why these words should always be capitalized?

Reply:
These are words that are always capitalized because they are commemorative dates.

Reply:
no, they aren't.

Reply:
Then I would expect there to be contexts in which they shouldn't be capitalized and thus this rule should specify some amount of context.

Reply:
after searching further, they are <np> when they are dates. So this part doesn't make sense